light-system-programs/programs/verifier_program_one/src/lib.rs

```diff
@@ -134,6 +134,8 @@ pub struct LightInstructionSecond<'info, const NR_CHECKED_INPUTS: usize> {
     pub signing_address: Signer<'info>,
     pub system_program: Program<'info, System>,
     pub program_merkle_tree: Program<'info, MerkleTreeProgram>,
+    #[account(mut)]
+    pub event_merkle_tree: AccountLoader<'info, EventMerkleTree>,
```

Comment:
this might mess with the indexer because it assumes the account sender and recipient accounts to be in certain places, better put it on the bottom of the account struct.

Reply:
Fixed, the order should be consistent now